daphne/src/testing.rs

```diff
@@ -554,9 +554,7 @@ impl DapLeader<BearerToken> for MockAggregator {
             .unwrap()
             .as_secs();
 
-        if collect_req.batch_interval.duration / task_config.min_batch_duration
-            > self.global_config.max_batch_duration
-        {
+        if collect_req.batch_interval.duration > self.global_config.max_batch_duration {
             return Err(DapError::Abort(DapAbort::BadRequest(
                 "batch interval too large".to_string(),
             )));
```

Comment:
Why do we need these checks in MockAggregator, now that they're implemented in the generic DAP logic?

Reply:
The interval check has to be implemented in both MockAggregator and Daphne Workers, as it is called during within the `init_collect_job` function.

Reply:
My point is: Why isn't this logic in `DapLeader::http_post_collect()` Instead?

Reply:
I've moved this to http_post_collect.